app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Validation\ValidationException;
+
+class UserController extends Controller
+{
+    public function register(Request $request): JsonResponse
+    {
+
+            $validatedData = $request->validate([
+                'name' => 'required|string|max:255',
+                'email' => 'required|email',
+                'password' => 'required|string|max:255',
+            ]);
+
+           dd($validatedData);
```

Comment:
here you need to call the registerUser method from UserServices and return a 200 status